parmed/rdkit/rdkit.py

```diff
@@ -0,0 +1,41 @@
+"""
+This package contains classes responsible for loading rdkit objects
+"""
+
+from __future__ import print_function, absolute_import
+from parmed.formats import PDBFile
+from parmed.utils.six.moves import StringIO
+
+
+class RDKit(object):
+
+    @staticmethod
+    def load(rmol):
+        """
+        Load a :class:`Mol` object and return a populated :class:`Structure`
+        instance
+
+        Parameters
+        ----------
+        rmol: :class:`Mol`
+            RDKit :class:`Mol` object to convert
+
+        Examples
+        --------
+        >>> from rdkit import Chem
+        >>> import parmed as pmd
+        >>> mol = Chem.MolFromSmiles('Cc1ccccc1')
+        >>> struct = pmd.load_rdkit(mol)
+        """
+        from rdkit import Chem
```

Comment:
this require absolute import since py2 will be confused about 'rdkit' in parmed and rdkit package (from omnia).
(added absolute_import)